.github/workflows/release-nightly.yml

```diff
@@ -19,20 +19,37 @@ jobs:
       build: ${{ matrix.build }}
 
   release:
-    name: Make Release on GitHub
+    name: Make Release on GitHub and pypi.anaconda.org
     needs: build_wheels
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Comment:
I would keep all versions of ubuntu the same across all CI? I think it's still 22.04 in other places.
That said, it's probably time to bump that version everywhere?